x-pack/spec/monitoring/inputs/metrics/stats_event_factory_spec.rb

```diff
@@ -91,7 +95,7 @@
    end
  end
 
- context "old model" do
+ xcontext "old model" do
```

Comment:
Let's add a comment here with the link to the issue, something like:

```suggestion
 # TODO: fix issue <link>
 xcontext "old model" do
```